projects/listening-comp/services/transcript-processor/app.py

```
@@ -17,6 +25,42 @@
+    use_remote_service=True,
+    device=DEVICE
+)
+
+audio_service = MicroService(
+    name="audio_service",
+    host="audio-module",  # Docker service name
+    port=5002,
+    endpoint="/api/extract-audio",
+    service_type=ServiceType.AUDIO,
+    use_remote_service=True,
+    device=DEVICE
```

Ruff (0.8.2)

Undefined name `DEVICE`

(F821)